pilot/pkg/networking/util/util.go

```diff
@@ -291,11 +291,6 @@ func IsIstioVersionGE13(node *model.Proxy) bool {
 		node.IstioVersion.Compare(&model.IstioVersion{Major: 1, Minor: 3, Patch: -1}) >= 0
 }
 
-// IsXDSMarshalingToAnyEnabled controls whether "marshaling to Any" feature is enabled.
-func IsXDSMarshalingToAnyEnabled(node *model.Proxy) bool {
-	return !features.DisableXDSMarshalingToAny
```

Comment:
Should also delete `features.DisableXDSMarshalingToAny` if we are making it do nothing?

Reply:
Would it break some command line flag? If so, we cannot easily remove it.

Reply:
No its just an environment variable. If we keep it around its confusing and actually shows up in our docs so its confusing to users as well

Reply:
OK, removed.